src/Microsoft.ML.OnnxTransformer/OnnxCatalog.cs

```diff
@@ -33,15 +33,15 @@ public static class OnnxCatalog
         /// Applies a pre-trained Onnx model.
         /// </summary>
         /// <param name="catalog">The transform's catalog.</param>
-        /// <param name="modelFile">The path of the file containing the ONNX model.</param>
         /// <param name="outputColumnName">The output column resulting from the transformation.</param>
         /// <param name="inputColumnName">The input column.</param>
+        /// <param name="modelFile">The path of the file containing the ONNX model.</param>
         /// <param name="gpuDeviceId">Optional GPU device ID to run execution on, <see langword="null" /> to run on CPU.</param>
         /// <param name="fallbackToCpu">If GPU error, raise exception or fallback to CPU.</param>
         public static OnnxScoringEstimator ApplyOnnxModel(this TransformsCatalog catalog,
-            string modelFile,
             string outputColumnName,
             string inputColumnName,
```

Comment:
>inputColumnName [](start = 19, length = 15)

should this default to the outputColumnName like the other inputColumns?

Reply:
my comment is the same. 

---
In reply to: [268883194](https://github.com/dotnet/machinelearning/pull/3086#discussion_r268883194) [](ancestors = 268883194)

Reply:
I went back and forth on this. I don't think we need to have this as an "in-place" transform because we expect models to create new columns, similar to the TensorFlow transform.